packages/react-components/src/stories/Toast.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+
+import {
+  IToastProps,
+  Toast as ToastComponent,
+  Variants,
+} from '../components/Toast';
+
+export default {
+  title: 'Components/Toast',
+  component: ToastComponent,
+} as ComponentMeta<typeof ToastComponent>;
+
+interface IToastArgs extends IToastProps {
+  label: string;
+}
+
+export const Toast = (args: IToastArgs): React.ReactElement => (
```

Comment:
I would also add here more stories/variants to keep the visual-test coverage.

Reply:
Toast stories are updated